test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractMethodIsolatedCloudIntegrationTest.java

```diff
@@ -32,7 +33,11 @@
 
     @Before
     public void initializeDeployment() {
-        deploymentScenario = createDeploymentScenario(deploymentScenarioFactory);
+        try {
+            deploymentScenario = createDeploymentScenario(deploymentScenarioFactory);
+        } catch (UnsupportedOperationException ex) {
+            Assume.assumeNoException(ex);
```

Comment:
Maybe it would be better to apply category, something like Apb class, to differentiate tests which can be executed with APB and which not. Handling exceptions in initialization code makes the test less readable and understandable.

Reply:
+1 for the category. This assume also do not work correctly, I found out that if assume is used in parent class or called from parent class, tests are not skipped.